l10n_it_reverse_charge/tests/rc_common.py

```diff
@@ -255,6 +282,14 @@ def _create_rc_type_taxes(cls):
             }
         )
 
+        cls.rc_type_tax_eeu_no_selfinvoice_extra = rc_type_tax_model.create(
```

Comment:
Questa non mi pare venga usata, si può rimuovere?

Reply:
Non si può perchè restituirebbe questo errore:
`odoo.exceptions.UserError: Can't find tax mapping for Tax 22% Purchase Extra-EU in Reverse Charge Type Extra EU (selfinvoice), please check the configuration.`

Se noti anche le altre `account.rc.type.tax` non sono usate direttamente, ma vengono richiamate indirettamente usando le mappature delle imposte che definiscono.

Reply:
:+1: chiaro grazie